src/dmclock/sim/src/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 include_directories(ssched) # ssched code
 include_directories(../../src) # dmclock code
 include_directories(../../support/src)
-include_directories(${BOOST_INCLUDE_DIR})
+include_directories(SYSTEM ${BOOST_INCLUDE_DIR})
```

Comment:
@adamemerson oh, i forgot this: https://github.com/ceph/ceph/blob/master/src/dmclock/README.before-modifying-files-here-or-below